eogrow/pipelines/testing.py

```diff
@@ -129,16 +149,17 @@ def get_execution_arguments(self, workflow: EOWorkflow) -> List[Dict[EONode, Dic
         """Extends the basic method for adding execution arguments by adding seed arguments a sampling task"""
         exec_args = super().get_execution_arguments(workflow)
 
-        add_feature_nodes = [
-            node
-            for node in workflow.get_nodes()
-            if isinstance(node.task, (DummyRasterFeatureTask, DummyTimestampFeatureTask))
-        ]
+        add_feature_nodes = list(self._nodes_to_configs_map)
         add_feature_nodes.sort(key=lambda node: node.get_name())  # To ensure seeds are always given in the same order
```

Comment:
would it work in one line if you used `sorted` here?

Reply:
Good point, I updated. I also renamed `node` -> `_node` because PyCharm was telling me `node` variable is already used in the outer scope.